assemblies/using-explainability.adoc

```diff
@@ -0,0 +1,24 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+:productname-long: Open Data Hub
+:productname-short: Open Data Hub
+
+:context: explainers
+
+[id="using-explainability_{context}"]
+= Using explainability
+
+As a data scientist, you might want to understand how your machine learning model makes its predictions and decisions. You can use explainers from TrustyAI to provide saliency explanations for model inferences in {productname-long}.
```

Comment:
I think we can simplify ",...you might want to understand..." with something else " ,... you can learn...", maybe.